Makefile

```diff
@@ -112,8 +115,9 @@ build-linux:
 	  -e GOOS=linux                                                                    \
 	  -e GOARCH=amd64                                                                  \
 	  -e GOPATH=/go                                                                    \
+	  -e GOFLAGS=$(GOFLAGS)                                                            \
```

Comment:
Is this flag not needed for `build-local`?

Reply:
Yes, but I have exported the `GOFLAGS` environment variable above for the local environment